Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/spark/common/ADLSGen2Deploy.java

```diff
@@ -75,7 +75,7 @@ private URI getUploadDir() {
         ADLSGen2FSOperation op = new ADLSGen2FSOperation(this.http);
         return op.createDir(dirPath)
                 .onErrorReturn(err -> {
-                    if (err.getMessage().contains(String.valueOf(HttpStatus.SC_FORBIDDEN))
+                    if (err.getMessage()!= null && err.getMessage().contains(String.valueOf(HttpStatus.SC_FORBIDDEN))
```

Comment:
Who generates the `null` message exception?

Reply:
org.apache.http.client.clientprotocolexception